packages/repository/src/__tests__/integration/repositories/relation.factory.integration.ts

```diff
@@ -564,11 +513,7 @@ class CustomerCartItemLink extends Entity {
   customerId: number;
   itemId: number;
   static definition = new ModelDefinition('CustomerCartItemLink')
-    .addProperty('id', {
-      type: 'number',
-      id: true,
-      required: true,
-    })
```

Comment:
This is actually an interesting discussion point. Should "through" models have their own primary key (`id`), or should we use a composite primary key composed from source & target keys (e.g. `itemId` + `customerId`)?

@raymondfeng What's your opinion?

Reply:
Imagine we have `Appointment` as the `through` model to connect `Doctor` and `Patient`, a patient may have multiple appointments with the same doctor. So `doctorId + patientId` won't be unique in `Appointment` model, which should have its PK. 

For the relation, we need to use two FKs (doctorId and patientId) instead of PK for `Appointment`.